__tests__/integrations/python.Dockerfile

```diff
@@ -18,7 +18,3 @@ ADD __tests__ /src/__tests__
 COPY --from=build-env /src /src
 # Pip installs it's packages in some global location 🤷‍♂️
 COPY --from=build-env /usr/local/ /usr/local/
-# Without these I get some error about shared libraries being missing:
-# python3: error while loading shared libraries: libpython3.10.so.1.0: cannot open shared object file: No such file or directory
-COPY --from=build-env /lib /lib
-ENV LD_LIBRARY_PATH=/lib:/usr/lib:/usr/local/lib
```

Comment:
With the Node 16 image these were also causing shared library issues, removing them seems to get the image working fine locally.

Reply:
Esh, yeah I spent a bunch of time debugging this to get it to work with Node.js 18 images. I nearly pulled my hair out with this 😬